.swiftlint.yml

```diff
@@ -81,7 +80,7 @@ redundant_nil_coalescing: error
 trailing_comma:
   mandatory_comma: true
 vertical_whitespace:
-  max_empty_lines: 1
+  max_empty_lines: 2
```

Comment:
Can we just remove this check entirely?

Reply:
Right here I’m just going for alignment with our internal config